openssl/src/x509/store.rs

```diff
@@ -176,6 +176,37 @@ impl X509LookupRef<HashDir> {
     }
 }
 
+/// Marker type corresponding to the [`X509_LOOKUP_file`] lookup method.
+///
+/// [`X509_LOOKUP_file`]: https://www.openssl.org/docs/man1.1.1/man3/X509_LOOKUP_file.html
+pub struct ASN1;
```

Comment:
Why is this called `ASN1` and not e.g. `File`?

Reply:
Thank you for pointing that out, that was a reference that I didn't replace with the appropriate name when creating the PR. I've updated the name to be `File` to match the `HashDir` struct above